Themis/Models/UML/UMLRelationshipType.swift

```diff
@@ -33,4 +33,16 @@ enum UMLRelationshipType: String, Decodable {
     case classDependency = "ClassDependency"
     case classAggregation = "ClassAggregation"
     case classComposition = "ClassComposition"
+    
+    /// The string that should be shown on the UML relationship to indicate the type of the element
+    var annotationTitle: String {
+        switch self {
+        case .useCaseExtend:
+            return "<<extend>>"
```

Comment:
I think the much-less/much-more unicode sign looks better here:
```suggestion
            return "≪extend≫"
```